prow/clonerefs/run.go

```diff
@@ -81,7 +82,24 @@ func (o Options) Run() error {
 		go func() {
 			defer wg.Done()
 			for ref := range input {
-				output <- cloneFunc(ref, o.SrcRoot, o.GitUserName, o.GitUserEmail, o.CookiePath, env, oauthToken)
```

Comment:
This always writes to the output channel. The changes need to keep doing that.

Reply:
I'm missing the fails-3-times scenario writing-to-channel. Will figure out.